internal/fuse/dir.go

```diff
@@ -133,6 +139,11 @@ func (d *dir) calcNumberOfLinks() uint32 {
 
 func (d *dir) ReadDirAll(ctx context.Context) ([]fuse.Dirent, error) {
 	debug.Log("ReadDirAll()")
+	err := d.Open()
+	if err != nil {
+		return nil, err
+	}
+	debug.Log("called")
```

Comment:
I think we can remove that `debug.Log` call

Reply:
yes - this was just to debug; is removed.